recipes/bmi-tester/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: bmi-tester
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms/bmi-tester/archive/v{{ version }}.tar.gz
+  sha256: 63ad43e89d96db72ec0c5a489a481835126dc4bad3701d3899cfd300e27df33e
+
+build:
+  number: 0
+  skip: True  # [win]
+  entry_points:
+    - bmi-test=bmi_tester.bmipytest:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
```

Comment:
Does this package need `setuptools` to get `pkg_resources:

https://github.com/csdms/bmi-tester/blob/master/bmi_tester/bmipytest.py#L7

It seems like it's pulling it in from somewhere else perhaps. Not sure if it should be an explicit dependency.

Reply:
@synapticarbors Thanks for looking at this!

It seems that `setuptools` (and so `pkg_resources`) comes along with python. Because of that, I've been leaving it out of my recipes but would you like me to add a `setuptools` requirement?